kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -766,12 +766,11 @@ def _init_cterm(init_term: KInner) -> CTerm:
     return init_cterm
 
 
-def get_final_accounts_cell(proof_digest: str, proof_dir: Path) -> KInner:
+def get_final_accounts_cell(proof_digest: str, proof_dir: Path) -> tuple[KInner, KInner]:
     apr_proof = APRProof.read_proof(proof_digest, proof_dir)
     target = apr_proof.kcfg.get_unique_target()
-    cover = single(apr_proof.kcfg.covers(target_id=target.id))
-    accounts_cell = get_cell(cover.source.cterm.config, 'ACCOUNTS_CELL')
-    return accounts_cell
+    cterm = single(apr_proof.kcfg.covers(target_id=target.id)).source.cterm
+    return (cterm.cell('ACCOUNTS_CELL'), cterm.cell('ACTIVEACCOUNTS_CELL'))
```

Comment:
I'm assuming that `cterm.cell(...)` is equivalent to `get_cell(cterm.config, ...)`, right?

Reply:
Yep! But it's better because once you've called `cterm.cell(...)` once for any cell, every subsequent call will be faster (it's memoized at the `CTerm` level).

Here is the implementation of `CTerm.cell`: https://github.com/runtimeverification/pyk/blob/d9d6e8c101bc48570ebfdc62bf7b851af468aa23/src/pyk/cterm.py#L103
And the implementation of `get_cell`: https://github.com/runtimeverification/pyk/blob/d9d6e8c101bc48570ebfdc62bf7b851af468aa23/src/pyk/kast/manip.py#LL528C6-L528C6

Almost always using the `CTerm` interface means faster access and better behavior overall.